src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediatePortalStatement.java

```diff
@@ -62,7 +62,7 @@ public short getParameterFormatCode(int index) {
   @Override
   public short getResultFormatCode(int index) {
     if (this.resultFormatCodes == null || this.resultFormatCodes.isEmpty()) {
-      return 0;
+      return super.getResultFormatCode(index);
```

Comment:
Bubble up to the super implementation. This does nothing at the moment, but we should implement a smarter default than just return 0 (text) when the client did not specify what it wants. That gives us the freedom to choose the binary format when we think that is better.